uploadNeuroDB/minc_insertion.pl

```diff
@@ -55,7 +55,7 @@
 my $create_minc_pics    = 0;   # Default is 0, set the option to overide.
 my $globArchiveLocation = 0;   # whether to use strict ArchiveLocation strings
                                # or to glob them (like '%Loc')
-my $no_nii      = 1;           # skip NIfTI creation by default
+my $create_nii  = 1;           # skip NIfTI creation by default
```

Comment:
is the fact that this default value changed intentional?

Reply:
oops. Good catch. I changed it once and then must have cancelled the change. Changing it to be 0 as it should be.

Reply:
@driusan Done!